node/src/service.rs

```diff
@@ -715,12 +722,16 @@ where
 	}
 
 	let announce_block = {
-		let network = network.clone();
-		Arc::new(move |hash, data| network.announce_block(hash, data))
+		let sync_service = sync_service.clone();
+		Arc::new(move |hash, data| sync_service.announce_block(hash, data))
 	};
 
 	let relay_chain_slot_duration = Duration::from_secs(6);
 
+	let overseer_handle = relay_chain_interface
```

Comment:
similar to Astar's and [Phala](https://github.com/Phala-Network/khala-parachain/blob/32ebbb0c204a114d522967c3cb795ad46202935f/node/src/service/mod.rs#L380-L382)